iroh-ctl/tests/cmd/peer_ids.trycmd

```diff
@@ -0,0 +1,6 @@
+```
+$ iroh-ctl p2p peers
+1AXRDqR8jTkwzGqyu3qknicAC5X578zTMxhAi2brppK2bB
+1Ag5LvC3vwsQicTy1dxNdd1xiNxLUS4Aic4NtdHBqAHD3j
+
```

Comment:
why is there an empty line?

Reply:
I was confused about that too. It's unfortunately a trycmd thing. It requires it and uses that in its own examples too. I think it's so it can separate multiple commands from each other in the same docstrings, but it's definitely ugly.

Reply:
I see, thanks